src/borg/remote.py

```diff
@@ -17,7 +17,8 @@
 from .helpers import bin_to_hex
 from .repository import Repository
 
-RPC_PROTOCOL_VERSION = 2
+RPC_PROTOCOL_VERSION = 3
+RPC_PROTOCOL_DOWNGRADABLE_VERSION = 2
```

Comment:
So do you want a `(min, max)` or a list? If the latter which item do we pull if the given version isn't supported?